pkg/plugin/tcpretrans/tcpretrans_linux.go

```diff
@@ -56,6 +58,12 @@ func (t *tcpretrans) Init() error {
 	// Create tracer. In this case no parameters are passed.
 	t.tracer = &tracer.Tracer{}
```

Comment:
I believe IG also requires `host.Init()` call before tracer is initialized; similar to https://github.com/microsoft/retina/blob/0dbffcbf9726710702121f71553d0af397d250b2/pkg/plugin/dns/dns_linux.go#L55 . Can you add that as well?

Reply:
OK, that's what i thought